sdk/search/azure-search-documents/pom.xml

```diff
@@ -91,6 +86,12 @@
       <version>1.18.1</version> <!-- {x-version-update;com.azure:azure-core-test;dependency} -->
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-core-serializer-json-jackson</artifactId>
+      <version>1.4.2</version> <!-- {x-version-update;com.azure:azure-core-serializer-json-jackson;dependency} -->
+      <scope>test</scope>
+    </dependency>
```

Comment:
If we don't include this, we fallback on `JsonSerializerProviders.createInstance(true)`, so, do we really need this dependency for testing?

Reply:
Yes, there are a few tests validating that a custom `JsonSerializer` is being used instead of the default `JsonSerializer` in `azure-core`. The main one is testing that `null` values are serialized and sent to the service.